src/operators/DerivativeConvolution.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+ * MRCPP, a numerical library based on multiresolution analysis and
+ * the multiwavelet basis which provide low-scaling algorithms as well as
+ * rigorous error control in numerical computations.
+ * Copyright (C) 2021 Stig Rune Jensen, Jonas Juselius, Luca Frediani and contributors.
+ *
+ * This file is part of MRCPP.
+ *
+ * MRCPP is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MRCPP is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with MRCPP.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * For information on the complete list of contributors to MRCPP, see:
+ * <https://mrcpp.readthedocs.io/>
+ */
+
+#include "DerivativeConvolution.h"
+#include "DerivativeKernel.h"
+#include "utils/Printer.h"
+
+namespace mrcpp {
+
+/** @returns New DerivativeConvolution object
+ *  @param[in] mra: Which MRA the operator is defined
+ *  @param[in] pr: Build precision, closeness to delta function
+ *  @details This will project a kernel of a single differentiated
+ *  gaussian with exponent sqrt(10/build_prec).
+ */
+template <int D>
+DerivativeConvolution<D>::DerivativeConvolution(const MultiResolutionAnalysis<D> &mra, double prec)
```

Comment:
Why derivative convolution? Is this computing a convolution integral after applying the derivative?

Reply:
It's the derivative operator written as a convolution with a differentiated delta function as kernel, approximated by a narrow p-type Gaussian. So "Convolution" indicates that it's derived from the `ConvolutionOperator` class, as opposed to the "real" derivative operators (`ABGVOperator`, `PHOperator` and `BSOperator`) which are not convolutions but `DerivativeOperators`. The `DerivativeConvolution` is not really useful, but it was an interesting take on our derivative problems of the past.

Reply:
OK, thanks for the explanation. Can you add this explanation to the Doxygen documentation string? 😸

Reply:
It's there in the header file:
https://mrcpp.readthedocs.io/en/latest/mrcpp_api/mwoperators.html#_CPPv4I_iEN5mrcpp21DerivativeConvolutionE

Reply:
Yes, but the comment sounds clearer to me than what is there right now and it also explains the difference with the other operators, including that this one is not really useful in comparison.